dart-sdk/pubspec.yaml

```diff
@@ -1,20 +1,20 @@
 name: eppo_sdk
 description: Eppo SDK for Dart
-version: '1.0.0'
+version: '0.1.0-alpha'
```

Comment:
thank you - a nice touch as we do some iteration to work out the packaging questions